pushy/src/test/java/com/eatthepath/pushy/apns/util/ApnsPayloadBuilderTest.java

```diff
@@ -611,6 +612,82 @@ void testBuildMdmPayload() {
         assertEquals("{\"mdm\":\"Magic!\"}", builder.buildMdmPayload("Magic!"));
     }
 
+
+    @Test
+    void testSetAlertSoundInLiveActivity() {
+        final String soundFileName = "dying-giraffe.aiff";
+        this.builder.setSound(soundFileName);
+        this.builder.setEvent(LiveActivityEvent.UPDATE);
+
+        final Map<String, Object> aps = this.extractApsObjectFromPayloadString(this.builder.build());
+        final Map<String, Object> alert = (Map<String, Object>) aps.get("alert");
+
+        assertEquals(soundFileName, alert.get("sound"));
+    }
+
+    @Test
+    void setTimestamp() {
+        final Instant timestamp = Instant.now();
+        this.builder.setTimestamp(timestamp);
+
+        final Map<String, Object> aps = this.extractApsObjectFromPayloadString(this.builder.build());
+
+        assertEquals(timestamp.getEpochSecond(), aps.get("timestamp"));
+    }
+
+    @Test
+    void setDismissalDate() {
+        final Instant dismissalDate = Instant.now();
+        this.builder.setDismissalDate(dismissalDate);
+
+        final Map<String, Object> aps = this.extractApsObjectFromPayloadString(this.builder.build());
+
+        assertEquals(dismissalDate.getEpochSecond(), aps.get("dismissal-date"));
+    }
+
+    @ParameterizedTest
+    @MethodSource("argumentsForSetEvent")
```

Comment:
Minor: I think you could simplify this by using [`@EnumSource`](https://junit.org/junit5/docs/current/user-guide/#writing-tests-parameterized-tests-sources-EnumSource) instead of `@MethodSource`.